bootstrap/templates/new/mix.exs

```diff
@@ -62,10 +62,15 @@ defmodule <%= app_module %>.Mixfile do
   # Specify target specific dependencies
   def deps("host"), do: []
   def deps(target) do
-    [{:nerves_runtime, "~> <%= runtime_vsn %>"},
-     {:"nerves_system_#{target}", "~> 0.10", runtime: false}]
+    [ system(target),
+      {:nerves_runtime, "~> <%= runtime_vsn %>"}
+    ]
   end
 
+  <%= for target <- targets do %>
+  def system("<%= target %>"), do: {:<%= String.to_atom("nerves_system_#{target}") %>, ">= 0.0.0", runtime: false}<% end %>
```

Comment:
Do you need to do the `String.to_atom` here? Since this is actually an EEx template, it's just text substitution, I think.